### PR TITLE
fixes #41 - Removed ByteBot reference

### LINKED ISSUE #41: remove notions of bytebot from CONTRIBUTING.md
Unnecessary references to project-specific parts for Bytebot should be removed.

### PR DESCRIPTION
**Pull request by Atlas34**

### CODE REVIEW

**Comment by mkzero** [comment]
There's still some mentions of external libraries that aren't used in this project and the Bytebot plugin system - could you remove those as well?

**Comment by Atlas34** [comment]
I will have a look

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -82,10 +82,6 @@ For the general code base, a clear style(with consideration for
 * imports must be at the top of the file
 * separate imports must be on separate lines
 * imports from the same module may be on a single line
-* imports must be grouped by:
-  * standard Python library
```
Comment (mkzero) [inline CONTRIBUTING.md]: The first two of this list were okay, actually, they can stay in

**Comment by mkzero** [comment]
I meant lines further down like `* plugin classes are all lowercase with no underscores` or lines mentioning libraries like Twisted. If you read on you'll probably find more in that file.

**Comment by Atlas34** [comment]
OK it reverted the changed for:
-* imports must be grouped by:
-  * standard Python library

and I removed the plugin line.

I think that all the references.

**Pull request merged**